broadcast/src/buffered/engine.rs

```diff
@@ -32,8 +32,6 @@ pub struct Engine<
     D: Digest,
     Cfg: CodecCfg,
     M: Digestible<D> + Codec<Cfg>,
-    NetS: Sender<PublicKey = P>,
```

Comment:
We don't know what network we'll use by the time we create `broadcast::buffered` in alto.